mlir/CMakeLists.txt

```diff
@@ -170,7 +170,7 @@ configure_file(
 #   The pybind11 library can be found (set with -DPYBIND_DIR=...)
 #   The python executable is correct (set with -DPython3_EXECUTABLE=...)
 # By default, find_package and probing for installed pybind11 is performed.
-# Super projects can set MLIR_DISABLE_CONFIGURE_PYTHON_DEV_PACKAGES=ON to 
+# Super projects can set MLIR_DISABLE_CONFIGURE_PYTHON_DEV_PACKAGES=ON to
```

Comment:
I'm not familiar with LLVM's review processes, but if it were my code base, I'd tell you "please make unrelated changes in a separate PR". It's trivial to review a PR that, say, fixes doc comments or typos, and it helps when reviewing and landing changes to keep PRs focused on one thing and one thing only. And it when looking, at, say, line-by-line blame to know whether a change came from a substantive PR or a trivial cleanup PR.

Reply:
Yes, I agree, this is a good practice in OSS
https://github.com/llvm/llvm-project/pull/121791